recipes/runjob/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt

```

Comment:
You can include this one liner in the meta.yml if you like.

Reply:
Ok, I will include this in the meta.yml file, Thanks.